cloudinit/net/openbsd.py

```diff
@@ -0,0 +1,43 @@
+# This file is part of cloud-init. See LICENSE file for license information.
+
+from cloudinit import log as logging
+from cloudinit import util
+import cloudinit.net.bsd
+
+LOG = logging.getLogger(__name__)
+
+
+class Renderer(cloudinit.net.bsd.BSDRenderer):
+
+    def write_config(self):
+        for device_name, v in self.interface_configurations.items():
```

Comment:
Is it typical to DHCP on all interfaces?  In clouds multi-nic DHCP will cause issues with default routing (if the DHCP server sends a gateway/route for each interface.  Certainly on Azure, secondary nics who DHCP will break networking going out of the box or hitting the IMDS due to traffic not flowing out of the primary NIC.

I strongly discourage dhcp'ing on all interfaces without a way to set a device or route metric.

Reply:
ok, I see. 

> I'd like to clarify expectations on dhcp on all nics, and one name fix up (net -> open).

Done, we don't default on DHCP anymore.